proposals/lip-0063.md

```diff
@@ -393,7 +412,7 @@ def createUserSubstoreArray():
             userSubstore.append(userObj)
     # append the legacy reserve account separately
     userSubstore.append(createLegacyReserveAccount())
-    sort userSubstore in the ascending order of (userObj.address + userObj.tokenID)
+    sort userSubstore in the lexicographical order of (userObj.address + userObj.tokenID)
```

Comment:
Could you please confirm, if the `userObj.address` here is in bytes or Lisk32 format? And, would the `userObj.tokenID` to be considered for the sorting only when there is a clash in the `userObj.address`?

Reply:
`userObj.address` is in bytes. And yes, sorting by `tokenID` is only done if there are two `userObj` instances with the same address.